function_maps.go

```diff
@@ -130,3 +131,13 @@ func getCompositeResource(req map[string]any) map[string]any {
 
 	return cr
 }
+
+func getExtraResources(req map[string]any, name string) []any {
+	var ers []any
+	path := fmt.Sprintf("extraResources[%s]items", name)
```

Comment:
Is this missing a `.` between the `]` and `name`?  If it works that's fine it just looks weird.